interview-questions.md

```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Are (most) Ruby methods accessors or mutators?
 
-  Your answer:
+  Your answer: I just discovered this recently in a review game with my classmates. Most Ruby methods are accessors. 
 
-  Researched answer:
+  Researched answer: I just discovered this recently in a review game with my classmates. Most Ruby methods are accessors. Accessor methods are needed for instance variables. With this, a variable is now referenced as behavior only one time and is no longer referenced directly many times throughout the program.
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: A block in Ruby consists of chunks of code. The block is invoked from a function.
 
-  Researched answer:
+  Researched answer: A block in Ruby consists of chunks of code and lies within "do" and "end". The code within the block is not put into effect at the time it is come up against. Ruby remembers where the block appears and then enters the method.
 
 
 
 3. What is a gem?
 
-  Your answer:
+  Your answer: A gem is a package manager that provides distributing of Ruby programs and libraries. I've had experience utilizing this in my most recent challenge questions in class.
 
-  Researched answer:
+  Researched answer: A gems is a package manager for Ruby that provides distributing of programs and libraries. It is a standard set in place to manage the installation of gems, and a server for distributing them.
 
 
 
 4. What is a relational database? Are there other kinds of databases?
 
-  Your answer:
+  Your answer: A relational database is a type of database that stores and gives access to data that are related. Each row in the table has its own ID called the key.
 
-  Researched answer:
+  Researched answer: A relational database is a type of database that stores and gives access to data that are related. Each row in the table has its own ID called the key. The columns contain attributes of the data which make it easy to display relationships between the data.
```

Comment:
This is good-- but relational databases go way deeper than this and I'd like to see your take on some of that info.